main.go

```diff
@@ -132,10 +141,32 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *graphiteAddr != nil {
+		go graphite.Graphite(metrics.DefaultRegistry, time.Nanosecond, *metricsPrefix, *graphiteAddr)
+	}
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: unable to get hostname: %s", err)
+		os.Exit(1)
+	}
+
+	metrics := metricsConfig{
+		url:      *metricsURL,
+		registry: metrics.DefaultRegistry,
+		prefix:   *metricsPrefix,
+		hostname: hostname,
+	}
+
+	if metrics.url != "" {
+		go metrics.publishMetrics()
+	}
+
 	status := newStatusHandler(dial)
 	if *statusPort != 0 {
 		mux := http.NewServeMux()
 		mux.Handle("/_status", status)
+		mux.Handle("/_metrics", metrics)
```

Comment:
I've decided that the status port should just publish `/_metrics` by default if it's enabled. The `metricsURL` flag is not for publishing at a URL, it's for posting the metrics periodically to a server to collect them.